tools/update-babel-eslint.sh

```diff
@@ -11,15 +11,16 @@ cd "$( dirname "${0}" )" || exit
 rm -rf node_modules/@babel
 mkdir babel-eslint-tmp
 cd babel-eslint-tmp || exit
-npm init --yes
+NPMCLI="../../deps/npm"
+node "$NPMCLI" init --yes
```

Comment:
```suggestion
$NPM init --yes
```

Reply:
Ideally $NPM would remain quoted because $PWD can contain spaces (even though using the Node.js repo from paths with spaces is already hard)